app/bundles/PageBundle/Translations/en_US/messages.ini

```diff
@@ -15,6 +15,8 @@ mautic.page.campaign.event.form.pages="Limit to Pages"
 mautic.page.campaign.event.form.pages.descr="Select the pages this trigger applies to. If none are selected, it'll apply to any page."
 mautic.page.campaign.event.form.url="URL"
 mautic.page.campaign.event.form.url.descr="Insert the URL of the page where you placed the tracking pixel. It is possible to use wildcart (*). Example: *product/123* will match http://web.com/product/1234.htm. Multiple URLs separate with comma."
+mautic.page.campaign.event.form.referer="Referrer"
+mautic.page.campaign.event.form.referer.descr="Insert the Referrer URL of the page where you placed the tracking pixel. It is possible to use wildcart (*). Example: *product/123* will match http://web.com/product/1234.htm. Multiple Referrer URLs separate with comma."
```

Comment:
@iwantaspeaker It doesn't make much sense to comment on a merged PR. But the typo will be fixed in https://github.com/mautic/mautic/pull/4067/commits/6fd1d747f4af10f4c15520e71b20a06ea69439f5